spyder/widgets/dependencies.py

```diff
@@ -145,7 +145,9 @@ def __init__(self, parent):
                               "them.<br><br>"
                               "<b>Note</b>: You can safely use Spyder "
                               "without the following modules installed: "
-                              "<b>%s</b> and <b>%s</b>")
+                              "<b>%s</b> and <b>%s</b>. Also note that new "

```

Comment:
Great! I like a lot the wording you decided to use. Just a couple of comments:

1. Add a `Please` at the beginning of the note:

        Please also note that new ....

2. Add this note in a new line by adding a `<br><br>` like this:

        "<b>%s</b> and <b>%s</b>.<br><br>"
        "Please also note that new ...."

That's all :-)